### PR TITLE
Unify migration descriptions

### LINKED ISSUE #16777: Unify migration descriptions
- Gitea version (or commit ref): 1.16.0+dev-138-gdb1e3d02a

## Description
Currently, there does not seem to exist a standard for the migration descriptions.
I, and most likely everyone else as well would prefer it, if these descriptions would be unified.
What do I mean by that?
1. I would replace `Migrating data …` with its more precise version `Migrate data …` as currently no action is already being performed
2. Some URLs are started with a capital letter, and some are not (example: `Github.com`, `notabug.org`). Either all URLs should start with their first letter capitalized, or none at all. In the first case: How do you handle urls with subdomains (especially `code.onedev.io`)? Do you capitalize all domain starting letters, or only the first?
3. Unify the overall descriptions: Currently, the unique descriptions each differ slightly enough to be noticeable. I think it would be better to use a unified approach for everything such as `Migrate data from <Provider.URL> or other <Provider> instances.`
4. Cut the `or Mirroring` from `Migrating or Mirroring git data from Git services` in `migrate.git.description`. All other migration descriptions also enable this and don't publicly announce it.

## Screenshots
image
Current appearance (in 1.15, not current dev 1.16)
image
Expected result afterwards

### LINKED ISSUE #16782: Gitea migration should migrate Pull Requests, not Merge Requests
- Gitea version (or commit ref): all up to 1.16.0+dev-138-gdb1e3d02a

## Description
Gitea does not use the term `Merge Request`.
It uses `Pull Request`. Yet, when migrating a gitea repo, we are presented with migrating `Merge Requests`.
Fix should be a one liner.

## Screenshots
image

### PR DESCRIPTION
**Pull request by delvh**
All migration descriptions now follow a unified naming approach.
Additionally, a Gitea migration now no longer asks whether to migrate `Merge requests`, but `Pull Requests` instead.
Fixes #16777.
Fixes #16782.

EDIT: I have just noticed that
`Migrate data from <Provider.URL> or another <Provider> instance.` can be used as well instead of
`Migrate data from <Provider.URL> or other <Provider> instances.`
Which one is preferred?

### New appearance
image

### Previous appearance
image

### CODE REVIEW

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -899,12 +899,12 @@ migrate.migrate = Migrate From %s
...
 migrate.migrating_failed = Migrating from <b>%s</b> failed.
 migrate.migrating_failed.error = Error: %s
-migrate.github.description = Migrating data from Github.com or Github Enterprise.
-migrate.git.description = Migrating or Mirroring git data from Git services
-migrate.gitlab.description = Migrating data from GitLab.com or Self-Hosted gitlab server.
-migrate.gitea.description = Migrating data from Gitea.com or Self-Hosted Gitea server.
-migrate.gogs.description = Migrating data from notabug.org or other Self-Hosted Gogs server.
-migrate.onedev.description = Migrating data from code.onedev.io or Self-Hosted OneDev server.
+migrate.github.description = Migrate data from github.com or other Github instances.
+migrate.git.description = Migrate data from any Git service.
```
Comment (noerw) [inline options/locale/locale_en-US.ini]:
```suggestion
migrate.git.description = Migrate a repository only from any Git service.
```
I think with git migrator it's worth pointing out that it targets different set of entities

**Review by noerw — APPROVED** [review]

**Review by KN4CK3R — APPROVED** [review]

**Comment by delvh** [comment]
> EDIT: I have just noticed that
> `Migrate data from <Provider.URL> or another <Provider> instance.` can be used as well instead of
> `Migrate data from <Provider.URL> or other <Provider> instances.`
> Which one is preferred?

This question hasn't been answered yet.

**Comment by lunny** [comment]
> > EDIT: I have just noticed that
> > `Migrate data from <Provider.URL> or another <Provider> instance.` can be used as well instead of
> > `Migrate data from <Provider.URL> or other <Provider> instances.`
> > Which one is preferred?
> 
> This question hasn't been answered yet.

Hm, for me. Both OK.

**Comment by KN4CK3R** [comment]
I vote for 1.

**Pull request merged**